src/pages/signin/SignInPage.js

```diff
@@ -69,12 +76,20 @@ class SignInPage extends Component {
         // - AND a validateCode has not been cached with sign in link
         const showLoginForm = !this.props.credentials.login && !this.props.credentials.validateCode;
 
+        // Show the Welcome form if
+        // - A user is signing up for a new account in a domain that is not controlled
```

Comment:
No need for the bullet here

```suggestion
        // Show the Welcome form a user is signing up for a new account in a domain that is not controlled
```